### PR TITLE
Update issue template

### PR DESCRIPTION
**Pull request by lauriejim**
Add bug and feature request template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,16 +1,34 @@
-**Node.js version**:
+<!-- Chose BUG or FEATURE REQUEST template and delete the one you don't use-->
+<!-- ⚠️ Don't forgot the checkbox at the end of your issue -->
 
-**npm version**:
+*BUG TEMPLATE* <!-- Delete this header from your issue -->
 
-**Strapi version**:
+<!-- ⚠️ Before write your issue be sure you are using :-->
```
Comment (soupette) [inline .github/ISSUE_TEMPLATE.md]: Before writing your issue make sure you are using

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,16 +1,34 @@
...
+<!-- Node 9.x.x -->
+<!-- npm 5.x.x -->
+<!-- Latest version of Strapi -->
```
Comment (soupette) [inline .github/ISSUE_TEMPLATE.md]: The latest

**Comment by abdonrd** [comment]
Maybe you might be interested: [Multiple issue and pull request templates](https://blog.github.com/2018-01-25-multiple-issue-and-pull-request-templates/).

**Comment by lauriejim** [comment]
Hi @abdonrd . The problem with this multiple template system is you can't chose the template from the issue UI of GitHub. And no body will use `?template=bug/feature_request` to chose an issue template.

**Comment by abdonrd** [comment]
@lauriejim pam! 💥

https://twitter.com/left_pad/status/992163857404645376

https://github.com/babel/babel/issues/new/choose

**Comment by derrickmehaffy** [comment]
Thats nice @abdonrd looks great from that example.

**Pull request merged**